spec/mongoid/validatable/uniqueness_spec.rb

```diff
@@ -2472,4 +2472,31 @@ class SpanishActor < EuropeanActor
       }.to raise_error(Mongo::Error::OperationFailure)
     end
   end
+
+  describe "i18n" do
+
+    context 'when using a different locale' do
+
+      around do |example|
+        I18n.with_locale(:fr) { example.run }
+      end
+
+      before do
+        # Translation key location is as per rails-i18n gem.
+        # See: https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/en.yml
+        I18n.backend.store_translations(:fr, { errors: { messages: { taken: 'est déjà utilisé(e)' } } })
+      end
+
+      after do
+        I18n.backend.instance_variable_get(:@translations).delete(:fr)
```

Comment:
I'm not thrilled with this, @Neilshweky what are your thoughts?

Reply:
I agree... As per removing a translation I think this would be a little better:
```suggestion
        I18n.backend.translations.delete(:fr)
```